GymBro/Views/HistoryViews/RoutineDetailView.swift

```diff
@@ -7,14 +7,156 @@
 
 import SwiftUI
 
+/// Detail for each routine
+///
+/// The view comes after `IntroHistoryView`
 struct RoutineDetailView: View {
+    
+    
+    // MARK: PROPERTY
+    
+    /// Mock ViewModel Singleton
+    @ObservedObject var vm = RDetailViewModel.instance
```

Comment:
It;s better to change to a more meaningful name. Somethin like detailViewModel, etc.

Reply:
Done rename!